backends/cuda/ceed-cuda-restriction.c

```diff
@@ -21,6 +21,9 @@
 static const char *restrictionkernels = QUOTE(
 
 #if __CUDA_ARCH__ < 600
+//------------------------------------------------------------------------------
```

Comment:
Note to myself: This `#if` doesn't work, doing it like this replaces the `atomicAdd` in all cases.